scripts/min_compatible_versions.py

```diff
@@ -14,7 +14,7 @@
 
 OUT_FILENAME = "min_compatible_versions.csv"
 OUT_DIRECTORIES = (".", "lib-injection")
```

Comment:
```suggestion
OUT_DIRECTORIES = (".", "lib-injection", "sources")
```

Sorry I missed this in previous PR of mine.